website/source/docs/guides/servers.html.md

```diff
@@ -18,28 +18,67 @@ That means if there are 3 server nodes, at least 2 must be available.
 In general, if you are ever adding and removing nodes simultaneously, it is better
 to first add the new nodes and then remove the old nodes.
 
-## Adding New Servers
+In this guide, we will cover the different methods for adding and removing servers.
 
-Adding new servers is generally straightforward. Simply start the new
+## Manually Add a New Server
+
+Manually adding new servers is generally straightforward. Simply start the new
```

Comment:
According to the internal style guide, avoid using "simply."

https://github.com/hashicorp/engineering-docs/blob/master/writing/style-guide.md#wordsphrases-to-avoid

Reply:
Updated!